Dockerfile

```diff
@@ -39,6 +39,11 @@ COPY docker/docker-entrypoint.sh docker-entrypoint.sh
 COPY startup_scripts/ /opt/netbox/startup_scripts/
 COPY initializers/ /opt/netbox/initializers/
 
+# Place default configuration in the container, so that it can be used
+# without docker-compose or older versions of docker-compose.yml
+RUN mkdir /etc/netbox
```

Comment:
this is not necessary according to the [Dockerfile reference][dockerfile]:

> * If `<dest>` doesn’t exist, it is created along with all missing directories in its path.

[dockerfile]: https://docs.docker.com/engine/reference/builder/#copy